### PR TITLE
docs: add community code of conduct

### PR DESCRIPTION
**Pull request by devadvocado**
Adding a community Code of Conduct file.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
```
Comment (kbarnard10) [inline CODE_OF_CONDUCT.md]: making -> make

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
```
Comment (gszr) [inline CODE_OF_CONDUCT.md]:
```suggestion
nationality, personal appearance, race, religion, or sexual identity, and
```

Comment (gszr) [reply CODE_OF_CONDUCT.md]: (As we are using the Oxford comma elsewhere, let's also use it here for consistency)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (devadvocado) [reply CODE_OF_CONDUCT.md]: done 👍

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (devadvocado) [reply CODE_OF_CONDUCT.md]: good catch, I agree. Updated to use Oxford comma

**Pull request merged**